src/bundle/Resources/public/js/scripts/helpers/context.helper.js

```diff
@@ -53,3 +55,9 @@ export const getPopper = () => Popper;
 export const getRouting = () => Routing;
 export const getTranslator = () => Translator;
 export const getRestInfo = () => restInfo;
+export const getRootDOMElement = () => rootDOMElement;
+export const isAppInStandaloneMode = () => {
```

Comment:
I'm not sure about using "standaloneMode" in admin-ui. Maybe something more generic with "origin" or "external instance"?